designs/2022-community-eslint-org/README.md

```diff
@@ -0,0 +1,360 @@
+Secondly, It could also put pressure on packages from other ESLint related
+organizations/maintainers to transfer their project to this new org, as it could
+be seen as the "correct" place for popular packages.
+
+## Backwards Compatibility Analysis
+
+<!--
+    How does this change affect existing ESLint users? Will any behavior
+    change for them? If so, how are you going to minimize the disruption
+    to existing users?
+-->
+
+It won't affect existing ESLint users, as package could be released under the
+same name on `npm`.  
+Even when we would release a package under the `@eslint-community` npm scope
+(for instance because we can't get the rights to publish to the existing npm
+package), people could still use the old package without any extra problems
+(except for the ones they already had) if they wanted to.
+
+## Alternatives
+
+<!--
+    What other designs did you consider? Why did you decide against those?
+
+    This section should also include prior art, such as whether similar
+    projects have already implemented a similar feature.
+-->
+
+- Private persons could create a fork of the above mentioned packages.  
+  This could however create ceveral well maintained forks, which would cause
+  fragmentation in the community.
+
+  More importantly, this could cause the "problem" of the fork being reliant on
+  that specific individual, which could put us in the same position
+  (abandoned/unmaintained package) all over again at some point in time.
+  Possibly having a new fork each X amount of time, would also have the burden
+  of needing to update all existing dependants each time.
+
+- The community could independently create multiple more focused organizations
+  to maintain a subset of ESLint related packages.  
+  Some examples of already existing organizations like that are
+  [@import-js](https://github.com/import-js) (which maintains
+  [`eslint-plugin-import`](https://github.com/import-js/eslint-plugin-import)),
+  [@jsx-eslint](https://github.com/jsx-eslint) (which maintains
+  [`eslint-plugin-jsx-a11y`](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y),
+  [`eslint-plugin-react`](https://github.com/jsx-eslint/eslint-plugin-react) &
+  [`jsx-ast-utils`](https://github.com/jsx-eslint/jsx-ast-utils)) &
+  [@standard](https://github.com/standard) (which maintains a couple of
+  [`standard`](https://github.com/standard/standard) related ESLint configs)
+
+## Open Questions
+
+<!--
+    This section is optional, but is suggested for a first draft.
+
+    What parts of this proposal are you unclear about? What do you
+    need to know before you can finalize this RFC?
+
+    List the questions that you'd like reviewers to focus on. When
+    you've received the answers and updated the design to reflect them,
+    you can remove this section.
+-->
+
+- What exact acceptance criteria do we want to use?
+- What's the main place of communication for the community core team?
+- Would projects be allowed to accept sponsorships?
```

Comment:
@nzakas If ESLint TSC is comfortable with it, these community packages could be included in the contributor pool.
This however has the implication that ESLint TSC needs to stay in the loop.

If that's not desirable, we could maybe set up sponsorship for the org & let the community core team decide each month who's in the community contributor pool?

Reply:
I’m not in favor of that. We need the organization to function on its own separate from the normal ESLint processes. We don’t have the bandwidth monitor an increasing number of packages every month, nor do we have the funding to support that.

I’d say that repos should be allowed to accept sponsorships. ESLint may choose to contribute as part of our ecosystem support efforts (like with typescript-eslint), but that wouldn’t be guaranteed.

Reply:
I don't think it's not a right way to have a sponsorship for all the projects. I can imagine the org can be very large, every project/team is allowed to accept its own sponsorship. e.g. the ts-eslint team should have its own funding/contributor-pool.

Reply:
@aladdin-add The @typescript-eslint org will still exist next to this new org.

I don't have a problem with each repo being able to setup its own sponsorship though. 🤷‍♂️
Just wanted to suggest a way of doing things that's in line with the core ESLint org.